en/features-guide/index.rst

```diff
@@ -0,0 +1,70 @@
+QGEP / TEKSI wastewater module
+===========
+
+TEKSI wastewater module (projet name: QGEP) is a selection of tools and a database implementation that allows you to:
+
+* manage and map your seewer network data with all its components in compliance with Swiss standards
+* indicate the characteristics of networks objects in the form of attributes such as diameter, material, depth, damages, construction year, etc.
+* produce plans and extract statistics from the database such as network values, total length of pipes, identification of future interventions, etc.
+* export geodata in compliance with Swiss standards
+
+Main features
+------------
+
+**Network objects**
+
+Creation, information and management of network elements according to the VSA-DSS model:
+
+* Wastewater structures
+* Special structures
+* Primary collectors
+* Secondary collectors (ex : private connection)
+* Catchments
+* GEP mesures
```

Comment:
I'm not sure, what ypu mean with this. I would mention the hydraulic modeling of overflows (pumps, weirs).

Reply:
General Wastewater Management Plan would be GWMP? I would add this maybe as separate functionnalities instead of having it in the Network objects section.

Same for Maintenances and Network inspection.

Reply:
So you would add a subject above network objects ?